rolling-shutter/p2p/metrics.go

```diff
@@ -24,7 +27,32 @@ var metricsP2PMessageHandlingTime = prometheus.NewHistogramVec(
 	[]string{"topic"},
 )
 
+var metricsP2PPeerTuples = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "p2p",
+		Name:      "peer_candidate",
+		Help:      "Collection of the encountered peer tuples.",
+	},
+	[]string{"peer_id", "peer_ip"})
+
+var metricsP2PPeerConnectedness = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "p2p",
+		Name:      "peer_connectedness",
+		Help:      "Collection of the connectedness (0=NotConnected; 1=Connected; 2=CanConnect; 3=CannotConnect) to a peer ID.",
+	},
+	[]string{"peer_id"})
```

Comment:
It might make sense to include our own id here. 
Or otherwise have a separate metric for the node's own id (but including it here probably makes querying easier).